### DISCUSSION TITLE
Should we keep waiting for 1st pull request's response, or create a new one for the next issue fix?

### DISCUSSION THREAD

**Discussion by ZuxingGit** [ZuxingGit/pygments#10] [Polls]
### Should we keep waiting for the 1st pull request's review, or create a new one for the next issuefix (#2 )?

**Comment by adluu2k1** [comment]
PRs are not related to each other. So I think it would be fine to create a new one if there is no response to the 1st one. We could still make changes to all PRs if required.

**Comment by Arseneme** [comment]
I think create a new one would be better.

**Comment by QSsoul** [comment]
Sorry for the wrong vote.  I think we should head to the new issue.
